src/styles/blocks/calendar__cell.scss

```diff
@@ -0,0 +1,15 @@
+.calendar__cell {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $cellSize;
+  width: $cellSize;
+  background-color: #eee;
+  border: 1px solid #000;
+  transition: 0.5s;
```

Comment:
list all transition properties, it is better for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x